File: test/fixtures/pyproject/src/calc.py
def classify(x, y):
    if x > 0:
        if y > 0:
            return "both"
        return "x only"
    elif y > 0 and x == 0:
        return "y only"
    return "neither"

def safe_div(a, b):
    try:
        return a / b
    except ZeroDivisionError:
        return None
    return "dead"

File: test/fixtures/jsproject-jest/src/calc.js
function classify(x, y) {
  if (x > 0) {
    if (y > 0) {
      return 'both';
    }
    return 'x only';
  } else if (y > 0 && x === 0) {
    return 'y only';
  }
  return 'neither';
}

function safeDiv(a, b) {
  try {
    if (b === 0) {
      throw new Error('div by zero');
    }
    return a / b;
  } catch (e) {
    return null;
  }
  return 'dead';
}

module.exports = { classify, safeDiv };
